src/codeflare_sdk/cluster/auth.py

```diff
@@ -82,7 +84,8 @@ def logout(self) -> str:
         """
         This function is used to logout of an OpenShift cluster.
         """
-        response = oc.invoke("logout")
+        args = [f"--token={self.token}", f"--server={self.server}:6443"]
```

Comment:
LGTM, just curious why the logout function needs the server/token?

Reply:
@Maxusmusti, while working on this I ran into a few edge cases where the logout function failed because the connection to the cluster had been interrupted. The addition of the token and server just makes the logout function more stable. 

If you don't like this we could replace it with some different error handling statements? Let me know what you think.